shell/src/main/java/tachyon/shell/command/DeleteLineageCommand.java

```diff
@@ -61,4 +61,9 @@ public void run(String... args) throws IOException {
   public String getUsage() {
     return "deleteLineage <lineageId> <cascade(true|false)>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Deletes a lineage specified by args. If cascade is specified as true, it deletes dependents lineages.";
```

Comment:
`Deletes a lineage. If cascade is specified as true, dependent lineages will also be deleted.`